src/test/java/org/broadinstitute/hellbender/tools/picard/sam/CreateSequenceDictionaryIntegrationTest.java

```diff
@@ -35,6 +35,22 @@ public void testBasic() throws Exception {
         IntegrationTestSpec.assertEqualTextFiles(outputDict, new File(BASIC_FASTA.getAbsolutePath() + ".dict"));
     }
 
+    @Test
+    public void testNoOutput() throws Exception {
+        final File expectedDict = new File(TEST_DATA_DIR, "basic.dict");
```

Comment:
Ah, I see -- ideally we shouldn't be creating temporary files within `src/test/resources` during tests. Might be preferable to copy the reference to a temporary location as returned by `createTempFile()`, and run the test from there.